javascript/Show/ShowView.jsx

```diff
@@ -86,65 +96,154 @@ export default class ShowView extends Component {
     }
   }
 
+  sortShow(inv, id) {
+    let showD = [...this.state.showData]
+    if (!inv) {
+      if (id == 5) {
```

Comment:
Is it possible if we can make these strings instead of numbers for clarity. Or maybe put some comment somewhere that indicates what these numbers mean.